### PR TITLE
Updated for last version of grunt-regarde (v0.1.1)

### PR DESCRIPTION
**Pull request by sleeper**
Get the list of files from grunt.regarde.changed.

### CODE REVIEW

File: tasks/livereload.js [hunk 1]
```diff
@@ -12,13 +12,26 @@ var server;
 
 module.exports = function (grunt) {
   grunt.registerTask('livereload', 'Inform the browser some files have changed', function () {
-    var files = this.args;
+    // var files = this.args;
```
Comment (sindresorhus) [inline tasks/livereload.js]: remove?

Comment (sleeper) [inline tasks/livereload.js]: Should be ok .. Sorry for this :(

## 

Frédérick Ros
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by sindresorhus** [comment]
Thanks for the quick fix :)

**Comment by sleeper** [comment]
Thx !! :two_hearts:

**Pull request merged**